CMakeLists.txt

```
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
 
+cmake_policy(SET CMP0144 NEW)
+cmake_policy(SET CMP0167 NEW)
```

```suggestion
if (POLICY CMP0167)
  cmake_policy(SET CMP0167 NEW)
endif()
```

Probably similar for `CMP0144`, but it's possible that cmake >= 3.20 already guarantees that.

---

Good point, thanks.  167 was added in 3.30.